browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -1138,6 +1142,14 @@ void BraveBrowserView::OnActiveTabChanged(content::WebContents* old_contents,
 
     // Setting nullptr doesn't detach the previous contents.
     UpdateContentsWebViewVisual();
+
+    if (need_to_update_secondary_web_view) {
+      // Revert back to default state.
+      contents_web_view_->SetFastResize(false);
+      secondary_contents_web_view_->SetFastResize(false);
+      contents_web_view_->DeprecatedLayoutImmediately();
```

Comment:
Are we worried about using these deprecated methods?

Reply:
I think `DeprecatedLayoutImmediately()` is exceptional case. The alternative way is calling `InvalidateLayout()` but it doesn't guarantee the view is laid out again.